src/mpv_calendar/scripts/tmux-service

```diff
@@ -37,6 +70,7 @@ is_session_running () {
 # Start the session and run the script
 start_session () {
     tmux new-session -d -s "$session_name" "$script_path" "${args[@]}"
+    date +%s > "$session_start_time"
```

Comment:
This implements window checking. If the service has been up for at least a few seconds, the window should have opened, and it should stay open for the duration of the service lifetime. Reset the timer every time the service restarts.